modules/lvms-monitoring-logical-volume-manager-operator.adoc

```diff
@@ -6,42 +6,56 @@
 [id="lvms-monitoring-using-lvms_{context}"]
 = Monitoring {lvms}
 
-When you use {rh-rhacm} to install {lvms}, you must configure {rh-rhacm} Observability to monitor all the clusters from one place.
+To enable cluster monitoring, you must add the following label in the namespace where you have installed {lvms}:
+[source,text]
+----
+openshift.io/cluster-monitoring=true
+----
+
+[IMPORTANT]
+====
+For information about enabling cluster monitoring in {rh-rhacm}, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html-single/observability/index[Observability] and link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html-single/observability/index#adding-custom-metrics[Adding custom metrics].
+====
 
 [id="lvms-monitoring-using-lvms-metrics_{context}"]
 == Metrics
 
-You can monitor {lvms} by viewing the metrics exported by the Operator on the {rh-rhacm} dashboards and the alerts that are triggered.
+You can monitor {lvms} by viewing the metrics.
 
-* Add the following `topolvm` metrics to the `allow` list:
-+
-[source,terminal]
-----
-topolvm_thinpool_data_percent
-topolvm_thinpool_metadata_percent
-topolvm_thinpool_size_bytes
-----
+The following table describes the `topolvm` metrics:
+
+.`topolvm` metrics
+[%autowidth,frame="topbot",options="header"]
```

Comment:
I like this table. I know "topbot" is valid a HTML component but [Asciidoctor](https://docs.asciidoctor.org/asciidoc/latest/tables/borders/) does not list it. Or platforms can be a bit funny when it comes to supporting cetain syntax. I do not want to kill your formatting buzz but asking a merge reviewer to check how platform docs render on Pantheon would be worthwile.